recipe/meta.yaml

```
@@ -43,8 +43,7 @@ requirements:
 
 test:
   commands:
-    - $R -e "library('data.table')"           # [not win]
-    - "\"%R%\" -e \"library('data.table')\""  # [win]
+    - Rscript -e "require('data.table'); data.table::test.data.table()"
```

Let's retain an independent loading test and stick to the syntax. Invocation with namespace is sufficient for `test.data.table()`:

```suggestion
    - $R -e "library('data.table')"                   # [not win]
    - $R -e "data.table::test.data.table()"           # [not win]
    - "\"%R%\" -e \"library('data.table')\""          # [win]
    - "\"%R%\" -e \"data.table::test.data.table()\""  # [win]
```

---

Ok no problem, I wasn't sure how best to set that up.

And I understand now why using variables `$R` / `%R%` that `conda-build` controls are preferable to looking up `Rscript` on `PATH`, sorry about that.